JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -405,8 +399,10 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
       return("")
     } else {
       hasSchar <- if (sidx == 1L) "" else "s"
-      footnote <- gettextf("The following item%s correlated negatively with the scale: %s. ",
-                           hasSchar, paste0(variables[idx], collapse = ", "))
+      footnote <- sprintf(ngettext(length(variables[idx]),
+                                   "The following item correlated negatively with the scale: %s. ",
+                                   "The following items correlated negatively with the scale: %s. "),
+                          paste(variables[idx], collapse = ", "))
```

Comment:
This footnote is shared between frequentist and bayesian right? Then you can again make a function and call the function from both files.

Reply:
well I just defined one function that checks the loadings now, and call it from both modules

Reply:
Yep and that's great! But the same principle applies to other code that is duplicated too (like this footnote): it's better to add it in a function and to call that function from different places. Otherwise there's a good chance that in the future when we change this footnote we forget to adjust it in both files.

Reply:
well, there is only one function which is .reliabilityCheckLoadings. But I made another function for the footnote. Which seems redundant to me, but I am here to obey. Also I got rid of other duplicate functions such as , checkErrors, and readData

Reply:
Yes, that is indeed redundant, hence my edit in my original post. I didn't realize the whole checkloading functions overlapped completely and could be merged. The footnote only needed to be in its own function when it was used in multiple locations, but that is no longer the case.

Very nice that you also merged the other duplicates!

Reply:
OK, I deleted the redundant function. All should be well